### ISSUE TITLE
download_wt103_model() does not seem to download the pretrained models.

### ISSUE THREAD

**Issue by nirmalsinghania2008** [fastai/fastai#916]
**Describe the bug**

`download_wt103_model() ` is not downloading the required files.

**To Reproduce**

Just run the example at https://github.com/fastai/fastai/blob/master/examples/text.ipynb

**Expected behavior**

download_wt103_model() should download required files itos_wt103.pkl and lstm_wt103.pth .

**Additional context**

I checked the code and downloaded the models from http://files.fast.ai/models/wt103_v1/ to get it to work.

**Comment by sgugger** [comment]
I just ran the notebook on a new clone and it did download the files. Could you give us the error message you saw?

**Comment by nirmalsinghania2008** [comment]
![error](https://user-images.githubusercontent.com/5922231/47177455-12afb580-d336-11e8-891f-e47386f319a2.JPG)
This was the error I faced.
But now something strange is happening,
Previously  `datasets.download_wt103_model() `took some time to execute, now it's executing in an instant when I try it again.

**Comment by sgugger** [comment]
If it doesn't take any time, it means the files are there (it tests to see if they exist and downloads only if they're not there). Can you try to remove the models folder, make sure you have the latest version of fastai, then run the notebook again?

**Comment by uptownnickbrown** [comment]
I just ran into I believe this same issue trying to run the text example notebook. I used the Developer Install instructions to install the most current code in this repo and I see my current version as `'1.0.8.dev0'`.

I resolved the issue by changing `datasets.download_wt103_model()` to `datasets.URLs.download_wt103_model()`. I think that's required since `download_wt103_model()` is now a class method on `URLs`, not at the `datasets` level. I believe this is the commit that caused that disconnect: https://github.com/fastai/fastai/commit/95f44e82b589c6985dcbe1178f144a0e36d36f0d#diff-48c59c669c350e1289260f46840bab8a

After that change the remainder of the notebook worked great.

I'm happy to submit that one-line change to fix this example notebook, but I'm not sure if that is the right fix, or if that method (and the other download methods) would be better moved back to `datasets.download_*` vs. `datasets.URLs.download_*`. I suspect there are other examples out there using `datasets.download_*` directly - this was my first test using fastai v1.
